hoot-core/src/main/cpp/hoot/core/conflate/highway/HighwayRfClassifier.cpp

```diff
@@ -298,10 +298,10 @@ void HighwayRfClassifier::_init() const
       _rf->importModel(file);
       file.close();
     }
-    catch (const Exception& e)
+    catch (const Exception&)
     {
       file.close();
-      throw e;
+      throw;
```

Comment:
Rethrowing the object is better accomplished with `throw` so that a copy of the exception isn't made and thrown.